src/ValiAddon.bas

```diff
@@ -136,12 +135,12 @@ Private Function getValiDict(Optional ByVal fetch_again As Boolean = False)
                 content(4) = Replace(vali("value"), ",", ".") & " " & vali("unit")
                 content(5) = Replace(vali("margin_plus"), ",", ".") & "%"
                 content(6) = Replace(vali("margin_minus"), ",", ".") & "%"
-                name_index = Int(InStr(vali("path"), "."))
-                If name_index Then
-                    content(7) = Replace(vali("path"), Left(vali("path"), name_index - 1), Project("name"))
-                Else
-                    content(7) = Project("name") & "." & vali("path")
-                End If
+                content(7) = vali("path")
+                'If name_index Then
```

Comment:
Why is this commented out? If it is left commented, please add a comment to explain why, otherwise just delete the block of code.

Reply:
Willl add a comment!